testdata/ede.tdir/ede.conf

```
@@ -0,0 +1,54 @@
+server:
+	verbosity: 2
+	interface: 127.0.0.1
+	port: @PORT@
+	use-syslog: no
+	directory: .
+	pidfile: "unbound.pid"
+	chroot: ""
+	username: ""
+	directory: ""
+	val-log-level: 2
+
+	# @TODO change this to local file
+	#auto-trust-anchor-file: "/var/lib/unbound/root.key"
+	trust-anchor-file: "bogus/trust-anchors"
```

It seems the TODO is now done
```suggestion
	trust-anchor-file: "bogus/trust-anchors"
```